style.css

```diff
@@ -0,0 +1,827 @@
+body {
+  color: #d3d3d3;
+  font-family: 'Circular-Loom', Arial, Helvetica, sans-serif;
+  margin: 0;
+  background-color: #efefef;
+  background-repeat: no-repeat;
+}
+
+.topBanner {
+  display: none;
+}
+
+.hero {
+  text-align: left;
+  position: inherit;
+}
+
+.backImage {
+  opacity: 0.05;
+
+  /* position: absolute; */
+
+  /* back image postion conflict with navmenu */
+  top: 0;
+  left: 0;
+  width: 100%;
+
+  /* height: 100%; */
+  z-index: -1;
+}
+
+h2 {
+  color: #000;
+}
+
+#novaLink {
+  text-decoration: none;
+  color: #f63d23;
+  text-align: center;
+  margin-left: 25vw;
+}
+
+#novaLink1 {
+  display: none;
+  text-decoration: none;
+  color: #f63d23;
+  text-align: center;
+  margin-left: 25vw;
+}
+
+h2.logo {
+  position: absolute;
+  left: 0;
+  margin-left: 15vw;
+}
+
+.backImageProd {
+  opacity: 0.1;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100%;
+  z-index: -1;
+
+  /* background-size: ; */
+}
+
+.storyH1 {
+  margin: 0;
+  width: 90vw;
+  font-size: 45px;
+  -webkit-background-clip: text;
+
+  /* background-clip: text; */
```

Comment:
- [ ] As the previous 2 code reviewers mentioned, kindly remove all the commented-out code since they can make your code looks messy and not professional.